controllers/dataprotection/backup_controller.go

```diff
@@ -217,6 +215,9 @@ func (r *BackupReconciler) handleNewPhase(
 
 	// set and patch backup status
 	if err = r.patchBackupStatus(backup, request); err != nil {
+		if intctrlutil.IsTargetError(err, dperrors.ErrorTypeWaitForExternalHandler) {
```

Comment:
Is it necessary? `patchBackupStatus()` never returns this error.

Reply:
return at ```
actions, err := request.BuildActions()
	if err != nil {
		return err
	}
```

Reply:
Got it. But why handle this error by requeuing?